documentation/src/main/asciidoc/topics/proc_configuring_global_persistent_location.adoc

```diff
@@ -0,0 +1,49 @@
+[id='configuring-global-persistent-location_{context}']
+= Configuring Global Persistent Location
+
+{brandname} preserves global state so that it can restore cluster topology and cached data after restart.
+If you enable global state in your {brandname} configuration, you should also configure a suitable location for {brandname} to persist global state.
+
+Global state is enabled by default with {brandname} Server, which persists cluster state to the `{server_home}/server/data` directory so it can perform graceful shutdowns.
+You do not need to configure a global persistent location for remote caches.
+
+For embedded caches, {brandname} defaults to the `user.dir` system property as the global persistent location, which is generally the directory where your application starts.
+You should always enable global state and configure a global persistent location for clustered embedded caches.
+
+.Procedure
+
+. Enable global state in one of the following ways:
++
+* Add the `global-state` element to your {brandname} configuration.
+* Call the `globalState().enable()` method in the `GlobalConfigurationBuilder` API.
++
+. Specify a global persistent location.
++
+* If the global persistent location is unique to each {brandname} instance, specify the `persistent-location` element or call the `persistentLocation()` method.
+* If the global persistent location is available to all {brandname} instances, specify the `shared-persistent-location` element or call the `sharedPersistentLocation()` method.
++
+.. Set a value for `path` to define the persistent location for global state.
++
+[TIP]
+====
+If you plan to use a file-based cache store for persistence, you can specify a directory on the host filesystem as the path.
+====
++
+.. Set a value for `relative-to` to configure the directory to which the path is relative.
```

Comment:
It might be nice to know when you would use relative-to and path or vice versa. It isn't completely clear to me, maybe @tristantarrant can fill in here as well?